com.unity.render-pipelines.universal/Runtime/Passes/ScriptableRenderPass.cs

```diff
@@ -82,6 +82,10 @@ public Color clearColor
             get => m_ClearColor;
         }
 
+        // Override to provide a custom name
+        protected internal virtual string profilingName { get => GetType().Name; }
```

Comment:
What would be the use case for that? Why would you change your profiling name at runtime?
It's can be overridden by derivation here, so that the sampler in parent (this) class can create it with the correct name in construction.
Any custom ScriptableRenderer should override this if auto generated name isn't good enough.

Reply:
Use  case: we have this RenderObjects pass. Users use it for general purpose (draw these objects with these settings) and want to name it. By having it like this 1) they can't override this for their custom passes 2) they can't change name for different instances.

 You can open this project to try out the profiler markers with custom passes: https://github.com/Unity-Technologies/UniversalRenderingExamples

Reply:
OK, I see. I'll try to fix this.

Reply:
They now are just protected sampler variables that you can set if necessary.